expo/tests/factories/gbe_factories.py

```diff
@@ -12,7 +12,7 @@ class Meta:
         model = conf.Conference
 
     conference_name = factory.Sequence(lambda n: "Test Conference %d" % n)
-    conference_slug = factory.Sequence(lambda n: u"test_conf %d" % n)
+    conference_slug = factory.Sequence(lambda n: u"test_conf_%d" % n)
```

Comment:
Thanks.  It broke in the test case I was writing in ticketing... and then I verified that slugs don't like whitespace. :)